recipe/meta.yaml

```diff
@@ -37,6 +37,8 @@ outputs:
       build:
         - {{ compiler("c") }}
         - {{ compiler("fortran") }}
+      run_constrained:
+        - {{ pin_subpackage("openblas", exact=True) }}
```

Comment:
Ah. In the issue you wrote: Add run_constrained for `libopenblas` on `openblas`, that's why I was confused!